src/app/about/about.component.html

```diff
@@ -1,3 +1 @@
-<p>
-  about works!
-</p>
+<img class="dev-cycle" width="600" [src]="devCycle">
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Add an "alt" attribute to this image. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://52.38.14.86:9000/sonar/coding_rules#rule_key=Web%3AImgWithoutAltCheck)